### ISSUE TITLE
Feature Request: support for gcp_private_service_connect_endpoint

### ISSUE THREAD

**Issue by rd-michel** [elastic/terraform-provider-ec#391]
Afaics theres is currently no support for gcp_private_service_connect_endpoint for resource type ec_deployment_traffic_filter

it is a requirement for the private service connect feature introduced in september
https://www.elastic.co/de/blog/secure-your-deployments-on-elastic-cloud-with-google-cloud-private-service-connect

is gcp private service connect already on the roadmap for the terraform provider?

**Comment by marclop** [comment]
Hi @rd-michel the feature was released as part of the `0.3.0` release, and it is documented accordingly: https://registry.terraform.io/providers/elastic/ec/latest/docs/resources/ec_deployment_traffic_filter#gcp-private-service-connect-type

```tf
locals {
  region = asia-east1
}

data "ec_stack" "latest" {
  version_regex = "latest"
  region        = local.region
}

# Create an Elastic Cloud deployment
resource "ec_deployment" "example_minimal" {
  # Optional name.
  name = "my_example_deployment"

  # Mandatory fields
  region                 = local.region
  version                = data.ec_stack.latest.version
  deployment_template_id = "gcp-storage-optimized"

  traffic_filter = [
    ec_deployment_traffic_filter.gcp_psc.id
  ]

  # Use the deployment template defaults
  elasticsearch {}

  kibana {}
}

resource "ec_deployment_traffic_filter" "gcp_psc" {
  name   = "my traffic filter name"
  region = local.region
  type   = "gcp_private_service_connect_endpoint"

  rule {
    source = "18446744072646845332"
  }
}
```

**Comment by Kushmaro** [comment]
Ah! thanks @marclop 
can this be closed then? I wasn't sure if we put only Azure in or this as well.
